websites/website/docs/guide/plugins-customization-path.md

```
@@ -0,0 +1,118 @@
+
+## Creating a reusable package
+
+### Using the CLI
+
+```bash
+npx @tsparticles/cli-create path <destination>
+```
+
+### Generated files
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)